packages/peregrine/lib/context/eventing.js

```diff
@@ -0,0 +1,57 @@
+import React, {
+    createContext,
+    useCallback,
+    useContext,
+    useMemo,
+    useRef
+} from 'react';
+import { Observable } from 'zen-observable-ts';
+
+const EVENT_NAME = 'beacon';
```

Comment:
This could be a prop, but due to the way [we render our context providers programmatically](https://github.com/magento/pwa-studio/blob/develop/packages/peregrine/lib/PeregrineContextProvider/peregrineContextProvider.js#L27-L29), there's no convenient way for people to actually supply a prop to `EventingContextProvider`.